src/CMakeLists.txt

```
@@ -393,7 +393,14 @@ if(ENABLE_OPENCV)
   find_package(OpenCV 4)
   if(NOT OpenCV_FOUND)
     set(ENABLE_OPENCV FALSE CACHE BOOL
-      "Build with OpenCV algorithms (requires Boost, Protobuf 3)" FORCE)
+      "Build with OpenCV algorithms (requires Protobuf 3)" FORCE)
+    # If we have version 4.5.1, all hope is lost
+  elseif(OpenCV_VERSION VERSION_EQUAL "4.5.1")
+    message(SEND_ERROR [[Incompatible OpenCV version detected
```

As I discovered in #664, `SEND_ERROR` level messages still cause the build-system generation to return a failure result. (Just, delaying it until the end of the script.) Oops. We don't want "wrong OpenCV version detected" to fail the `cmake` run, just disable OpenCV.

```suggestion
    message(WARNING [[Incompatible OpenCV version detected
```